router/path.go

```diff
@@ -0,0 +1,15 @@
+// +build !hosted
+
+package router
+
+func extractPath(host, path string) string {
+	return path
+}
+
+func systemPathFromSpace(space string) string {
+	return "/"
+}
+
+func systemPathFromPath(space string) string {
+	return "/"
```

Comment:
Make `/` into a keyword somewhere in `router` package, something like `BASEPATH`? We do use it a lot throughout.

Reply:
Yeah, sure I can do that.

Reply:
Do you think that this should be used only in this file or also in `path_hosted.go`?